### PR TITLE
fixes freebsd platform triplet name

### PR DESCRIPTION
**Pull request by noeffn**
Use freebsd for the triplet instead of bsd

### CODE REVIEW

**Comment by noeffn** [comment]
Added a platform mapping to make it easier for the future

File: xmake/modules/package/manager/vcpkg/find_package.lua [hunk 1]
```diff
@@ -73,10 +73,9 @@ function _find_package(vcpkgdir, name, opt)
     local arch = opt.arch
     local plat = opt.plat
     local mode = opt.mode
-    if plat == "macosx" then
-        plat = "osx"
-    end
-    arch = configurations.arch(arch)
+	
+    plat = configurations.plat(plat)
+	arch = configurations.arch(arch)
```
Comment (waruqi) [inline xmake/modules/package/manager/vcpkg/find_package.lua]: incorrect indent

File: xmake/modules/package/manager/vcpkg/install_package.lua [hunk 2]
```diff
@@ -47,10 +47,8 @@ function _install_for_classic(vcpkg, name, opt)
     local arch = opt.arch
     local plat = opt.plat
     local mode = opt.mode
-    if plat == "macosx" then
-        plat = "osx"
-    end
-    arch = configurations.arch(arch)
+    plat = configurations.plat(plat)
+	arch = configurations.arch(arch)
```
Comment (waruqi) [inline xmake/modules/package/manager/vcpkg/install_package.lua]: and here

File: xmake/modules/package/manager/vcpkg/install_package.lua [hunk 3]
```diff
@@ -80,12 +78,8 @@ function _install_for_manifest(vcpkg, name, opt)
...
     local arch = opt.arch
     local plat = opt.plat
-    if plat == "macosx" then
-        plat = "osx"
-    elseif plat == "bsd" then
-       plat = "freebsd"
-    end
-    arch = configurations.arch(arch)
+    plat = configurations.plat(plat)
+	arch = configurations.arch(arch)
```
Comment (waruqi) [inline xmake/modules/package/manager/vcpkg/install_package.lua]: here

Comment (noeffn) [reply xmake/modules/package/manager/vcpkg/install_package.lua]: already fixed in https://github.com/xmake-io/xmake/pull/2773/commits/b65703e6b570c00629f8f2431ca329c286290c9c

**Pull request merged**